components/Home/Hero/HeroSection.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Image from "next/image";
+
+export default function HeroSection() {
+  return (
+    <div className="flex px-6 justify-evenly py-9 flex-col md:flex-row items-center hero-section">
+      <div className="md:items-start  items-center  flex flex-col justify-center text-center md:text-start ">
+        <h1 className="font-semibold md:text-6xl lg:text-7xl text-5xl md:text-start  text-center	 ">
+          A perfect <span className="inline-block  md:hidden">road</span>map
```

Comment:
[Good to have]Can we create a constant file for all the texts? In that way, it is much easier to edit and update.

Reply:
that's cool